pxtblocks/plugins/functions/fields/fieldArgumentEditor.ts

```diff
@@ -1,21 +1,18 @@
 import * as Blockly from "blockly";
 import { FunctionManager } from "../functionManager";
+import { ArgumentEditorBlock } from "../blocks/argumentEditorBlocks";
 
 export interface FieldArgumentEditorConfig extends Blockly.FieldTextInputConfig {
```

Comment:
Do we still need this as a separate interface if it no longer adds something to the Blockly.FieldTextInputConfig?

Reply:
no, but maybe someday!